app/dashboard/notifications.py

```diff
@@ -213,7 +213,7 @@ def build_message_for_integration(bounty, event_name):
         pass  # no USD conversion rate
 
     title = bounty.title if bounty.title else bounty.github_url
-    msg = f"*{humanize(event_name.replace('bounty', 'funded_issue'))}*" \
+    msg = f"*{humanize_event_name(event_name))}*" \
```

Comment:
I hate this `event_name` duplication but I couldn't come up with a better name :disappointed: